### PR TITLE
reoroganize files into os, pro, and enterprise folders

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

Moves easily rip-outable pro functionality into the pro folder and creates an enterprise folder/liscence

# Testing

Please provide a list of the ways you can "access" or use the functionality. Please try and be exhaustive here, and make sure that you test everything you list.

- [ ] I have tested this on real data that is reasonable and large
- [ ] If I changed the interaction with JupyterLab, I tested that it does not break other programs (like VS Code), and tested that it works "multiple times" in the same notebook.

# Documentation

Note if any new documentation needs to addressed or reviewed.

### CODE REVIEW

File: mitosheet/src/enterprise/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+The Mito Enterprise license (the “Enterprise License”)
+Copyright (c) 2022 Saga Inc.
+
+With regard to the Mito Software:
+
+This software and associated documentation files (the "Software") may only be
+used in production, if you (and any entity that you represent) have agreed to,
+and are in compliance with, the Mito Subscription Terms of Service, or other
+agreement governing the use of the Software, as agreed by you and Mito,
+and otherwise have a valid Mito Pro license or Mito Enterprise license for the
```
Comment (aarondr77) [inline mitosheet/src/enterprise/LICENSE.txt:10]: Remove Pro

File: mitosheet/src/enterprise/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+correct number of user seats. Subject to the foregoing sentence, you are free to
+modify this Software and publish patches to the Software. You agree that Mito
+and/or its licensors (as applicable) retain all right, title and interest in and
+to all such modifications and/or patches, and all such modifications and/or
+patches may only be used, copied, modified, displayed, distributed, or otherwise
+exploited with a valid Mito Pro or Mito Enterprise license for the correct
```
Comment (aarondr77) [inline mitosheet/src/enterprise/LICENSE.txt:16]: Remove Pro

File: mitosheet/src/components/Mito.tsx [hunk 3]
```diff
@@ -56,8 +56,8 @@ import MeltTaskpane from './taskpanes/Melt/MeltTaskpane';
 import MergeTaskpane from './taskpanes/Merge/MergeTaskpane';
 import PivotTaskpane from './taskpanes/PivotTable/PivotTaskpane';
 import SplitTextToColumnsTaskpane from './taskpanes/SplitTextToColumns/SplitTextToColumnsTaskpane';
-import SetDataframeFormatTaskpane from './taskpanes/SetDataframeFormat/SetDataframeFormatTaskpane';
-import ConditionalFormattingTaskpane from './taskpanes/ConditionalFormatting/ConditionalFormattingTaskpane';
+import SetDataframeFormatTaskpane from '../pro/SetDataframeFormat/SetDataframeFormatTaskpane';
```
Comment (aarondr77) [inline mitosheet/src/components/Mito.tsx:59]: Add a taskpanes folder in the pro folder

File: mitosheet/mitosheet/pro/utils.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import json
+from typing import Any, Dict, List, Optional
+import pandas as pd
+from mitosheet.types import ConditionalFormattingCellResults, ConditionalFormattingInvalidResults, ConditionalFormattingResult, StateType
+from mitosheet.utils import MAX_ROWS, NpEncoder
+
+
+def get_conditonal_formatting_result(
```
Comment (naterush) [inline mitosheet/mitosheet/pro/utils.py:8]: This file can be named `conditional_formatting_utils.py`

**Review by naterush — COMMENTED** [review]
There is a LICENSE you need to update at the root of the monorepo, noting the enterprise license. Please update that.

Also, minor movement:
1. Move `mitosheet/pro/set_dataframe_format.py` to `mitosheet/pro/step_performers/set_dataframe_format.py`
2. Same with 1 hot encoding in enterprise

**Pull request merged**